packages/core/overlays/core-overlays-browser-internal/src/flyout/flyout_service.tsx

```diff
@@ -75,43 +75,56 @@ export class FlyoutService {
   public start({ analytics, i18n, theme, targetDomElement }: StartDeps): OverlayFlyoutStart {
     this.targetDomElement = targetDomElement;
 
-    return {
-      open: (mount: MountPoint, options: OverlayFlyoutOpenOptions = {}): OverlayRef => {
-        // If there is an active flyout session close it before opening a new one.
-        if (this.activeFlyout) {
-          this.activeFlyout.close();
+    const render = (
+      renderNode: () => React.ReactNode,
+      options: OverlayFlyoutOpenOptions
+    ): OverlayRef => {
```

Comment:
So, the whole purpose of `MountPoint` was to keep Core browser-side services framework agnostic. 

This PR is basically breaking that Core concept by introducing react as a first class citizen of Core (and breaking compatibility for other framework, and therefore potentially for 3rd party plugins). 

I'm not saying this is necessarily wrong (IHMO this framework agnostic thing was a bad decision), however this is not the kind of decision to take lightly. 

May we get more concept on what drove this decision, and who was included in the discussions?

cc @lukeelmers @jloleysens for visibility.

Reply:
Yeah it would be great if we could get some background on this -- without a PR description or a link to an issue I have zero context on how we got here.

Reply:
We are discussing it in SharedUX, will loop you in if we come to a consensus. Sorry, for GitHub to automatically pinging you, one part of my brain thought that overlays are owned by SharedUX, same as Chrome UI.